src/Sentry.Unity/Sentry.Unity.csproj

```diff
@@ -1,19 +1,16 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <OutDir>$(UnityPackageAssetsDirectory)/Plugins/Sentry</OutDir>
-    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <None Update="link.xml">

```

Comment:
We'll need this file included in the output dir, is it copied there even without this?

Reply:
This part I didn't understand. What is it for?